distributed/tests/test_stress.py

```diff
@@ -166,3 +166,25 @@ def test_stress_communication(c, s, *workers):
 
     result = yield future._result()
     assert isinstance(result, float)
+
+
+@slow
+@gen_cluster(client=True, ncores=[('127.0.0.1', 1)] * 8, timeout=10000)
```

Comment:
No, it's on the order of minutes on my machine.  It also doesn't yet actually find any issues though.  This is still a work in progress.